AGP-1H2.md

```diff
@@ -0,0 +1,43 @@
+---
+AGP: tbd
+Title: Delegation Frontend for Voting + Dandelion Voting
+Author: @lkngtn (@1hive)
+Status: Stage II
+Track: Finance
+Created: 2019-11-12
+---
+
+# **AGP-TBD: Delegation Frontend for Voting + Dandelion Voting**
+
+## **Address of the transfer recipient**
+
+ `0x9213737fbb72d8f924fca55d508ae2c004413d64` Vault of the [dandelion project organization](https://mainnet.aragon.org/#/1hive
+
+## **Amount of the transfer**
+
+20,000 DAI Due upon approval
+10,000 ANT Due upon successful delivery
+
+## **Number and frequency of transfers if recurring**
+
+2 (One upon approval, one upon successful delivery).
+For the purpose of budgeting, the full amount should be debited against this quarters budget.
+
+## **Purpose of the transfer**
+
+We believe that delegation has the potential to organically increase voter turnout by enabling less active participants to delegate their voting authority, and more active participants to engage more vocally in order to attract delegates.
+
+It has also been a long requested feature both for the ANT holders to use in the AGP process, as well as by users of the Aragon platform.
+
+In exchange for funding we will deliver the following as project deliverables:
+* We will create a frontend for principles to manage their delegation preferences, and for delegates to vote on behalf of their principles.
+* We will submit a PR with these changes and work with A1 to ensure it can be merged ahead of ANV6.
```

Comment:
not sure if i understood, the delegation will be a functionality of the dandelion voting app? or also for the normal voting app?

Reply:
The DV functionality is already implemented in the original Voting https://github.com/aragon/aragon-apps/pull/881/files. We would be implementing the frontend of this new functionality  for the original voting app and then copying what we have done into the Dandelion Voting.